project/profiles/capacity4more/modules/c4m/content/c4m_content_document/c4m_content_document.module

```diff
@@ -24,6 +24,59 @@ function c4m_content_document_c4m_og_vocab_info_content() {
 }
 
 /**
+ * Author autocomplete callback.
+ */
+function _edit_author_autocomplete($group_type, $gid, $string) {
+  extract(og_context());
+  $query = db_select('users', 'u');
+  $query
+    ->condition('u.uid', 0, '<>')
```

Comment:
@ordavidil Yes, it would get us all active group members.
Should I call this function and then use PHP for matching with the search string instead of the MySQL query?